go.mod

```diff
@@ -5,118 +5,143 @@ go 1.17
 replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
 
 require (
-	github.com/cosmos/cosmos-sdk v0.45.1
-	github.com/cosmos/ibc-go/v3 v3.0.0
+	github.com/cosmos/cosmos-sdk v0.46.0-rc1
+	github.com/cosmos/ibc-go/v3 v3.0.0-alpha1.0.20220530074104-c7c115244bad
```

Comment:
Does this refer to [this ibc-go branch](https://github.com/cosmos/ibc-go/tree/carlos/upgrade-sdk-0.46-tendermint-0.35)?

Reply:
No, actually no. It is pointing to a branch that is now merged into that one, so I do need to update this PR. Thanks for pointing this out!